src/rawchannel.cc

```diff
@@ -94,6 +96,7 @@ class RawChannel : public Nan::ObjectWrap
     Nan::SetPrototypeMethod(tpl, "start",           Start);
     Nan::SetPrototypeMethod(tpl, "stop",            Stop);
     Nan::SetPrototypeMethod(tpl, "send",            Send);
+    Nan::SetPrototypeMethod(tpl, "sendFD",          SendFD);                  //add dedicate function for SEND_CANFD frame
```

Comment:
comment is useless, please remove, the public interface.

Reply:
done